lib/ecs_deploy/capistrano.rb

```diff
@@ -186,4 +187,46 @@
       end
     end
   end
+
+  task increase_instances: [:configure] do
+    configs = fetch(:ecs_instance_fluctuation_manager_configs, [])
+    unless configs.empty?
+      regions = Array(fetch(:ecs_region))
+      regions = [EcsDeploy.config.default_region] if regions.empty?
+      regions.each do |region|
+        configs.each do |config|
+          logger = config.fetch(:logger, EcsDeploy.logger)
+          m = EcsDeploy::AutoScaler::InstanceFluctuationManager.new(
+            region: config[:region] || region,
+            cluster: config[:cluster] || fetch(:ecs_default_cluster),
+            cluster_to_asg: config[:cluster_to_asg],
+            desired_capacity: config[:desired_capacity],
+            logger: logger
+          )
+          m.increase
+        end
+      end
+    end
+  end
+
+  task decrease_instances: [:configure] do
```

Comment:
I prefer more meaningful name like  `set_desired_capacity_back_to_baseline_capacity`.

Reply:
This task will work like the followings:
* Set desired_capacity to baseline capacity (in configuration)
* Draining ECS container instances
* Stop ECS tasks
* Terminate ECS container instances step by step considering AZ balance

How about `terminate_redundant_instances` or `terminate_redundant_instances_step_by_step`?

Reply:
I think `terminate_redundant_instances` is good. 👍

Reply:
Fixed in 800c8b8